pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Work Ticket
+JIRA: https://cm-jira.usa.gov/browse/XX-XXXX
+Smartsheet: XXX0000000
+
+### Changes
+
+-   Add a description of your changes here
+
+### PR Checklist:
+
+1. [ ] Have you linted and tested your code locally prior to submission?
+2. [ ] Have you tagged the appropriate dev(s) for review?
+3. [ ] Have you moved any Jira/Smartsheet tickets to Code Review?
```

Comment:
1. does this need to be in a `.github` directory?
2. can we maybe add it in a separate PR?